packages/chord-mark-themes/scss/themes/print1.scss

```diff
@@ -1,7 +1,7 @@
 @import '../abstract';
 @import './common';
 
-.cmTheme-print {
+.cmTheme-print1 {
```

Comment:
this is a breaking change as all client code using this theme would have to update. I'm not aware of any application beside Chord Chart Studio but maybe it's better to avoid it?
Although I'm thinking that in any case, the markup change will also be a breaking change...